home/templates/home/index.html

```diff
@@ -3,10 +3,88 @@
 
 {% block content %}
 
-    <!-- TO DO - help wanted -->
-    <h1>TO DO : home > templates > home > index.html</h1>
-    <a class="btn btn-ci" href="{% url 'home' %}">Sample Button</a>
-    <br>
-    <small>custom class: <strong>btn-ci</strong></small>
+   <div class="container-fluid">
+    <div class="row">
+        <div class = "col-6 col-sm-12  col-md-6">

```

Comment:
check the extra spaces here, they're not needed.
There are trailing spaces on several places, you might consider putting the HTML code into an online code-formatter, just to get it formatted/cleaned up.
Also, is this correct or a bit redundant?
`col-6 col-sm-12 col-md-6` --- do we need 6 twice here?